cluster/gce/windows/smoke-test.sh

```diff
@@ -458,9 +458,9 @@ function test_linux_pod_to_internet {
   echo "TEST: ${FUNCNAME[0]}"
   local linux_command_pod
   linux_command_pod="$(get_linux_command_pod_name)"
-  local internet_ip="8.8.8.8"  # Google DNS
+  # A stable (hopefully) HTTP server provided by Cloudflare.
```

Comment:
fyi "curl https://8.8.8.8" works, 8.8.8.8 stopped serving http

Reply:
Ah, good to know :) I'll still leave this, using a non-Google server is also beneficial because GCP does weird network routing for google.com traffic.